src/cli.ts

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+import program = require("commander");
+import { parseOpenRPCDocument } from "@open-rpc/schema-utils-js";
+import { OpenRPC } from "@open-rpc/meta-schema";
+import { promisify } from "util";
+import { writeFile } from "fs";
+import { ensureDir, emptyDir } from "fs-extra";
+import MethodTypings, { languages, TLanguages } from "./";
+
+const cleanBuildDir = async (destinationDirectoryName: string): Promise<any> => {
+  await ensureDir(destinationDirectoryName);
+  await emptyDir(destinationDirectoryName);
+};
+
+const writeFilePromise = promisify(writeFile);
+const version = require("../package.json").version; // tslint:disable-line
+
+program
+  .version(version, "-v, --version")
+  .option(
+    "-d, --document [openrpcDocument]",
+    "JSON string or a Path/Url pointing to an open rpc schema",
+    "./openrpc.json",
+  )
+  .option(
+    "-o --output [directory]",
+    "typings output destination",
+    "/openrpc_typings",
+  )
+  .action(async () => {
+    let openrpcDocument: OpenRPC;
+    try {
+      openrpcDocument = await parseOpenRPCDocument(program.document);
+    } catch (e) {
+      console.error(e.message); // tslint:disable-line
+      console.error("Please revise the validation errors above and try again."); // tslint:disable-line
+      return;
+    }
+ 
+    const methodTypings = new MethodTypings(openrpcDocument);
+    await methodTypings.generateTypings()
+    await Promise.all(languages.map(async (language: TLanguages) => {
+      const destinationDirectoryName = `${process.cwd()}/${program.output}`;
+      await cleanBuildDir(destinationDirectoryName);
+
+      await writeFilePromise(
+        `${destinationDirectoryName}/${language}`,
+        methodTypings.getAllUniqueTypings(language),
```

Comment:
```suggestion
        methodTypings.toString(language),
```
based on the changes that are in master

Reply:
Thanks. Do you need this now I'll work on it?